### ISSUE TITLE
Accessibility Bug: The dialog header ‘The History of Pineapple’ and the content inside the dialog is not read out as soon as the dialog opens.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3117]
Fundamental core component:"Dialog"

Description: The dialog header ‘The History of Pineapple’ and the content inside the dialog is not read out(Screen Reader) as soon as the dialog opens

Expected: The dialog header and the content inside the dialog should be conveyed when a dialog is opened.

Attached the screenshot.

![image](https://user-images.githubusercontent.com/32538291/91553558-3ccb5900-e94b-11ea-9320-5e5e3984769b.png)

Note: - Issue persists for all the dialogs present within the ‘Dialog’ control.

**Comment by Sanjay-1312** [comment]
![image](https://user-images.githubusercontent.com/78714518/119811825-24519480-bf05-11eb-94b0-1be5e04c309e.png)
Hello Colleagues,

The dialog name followed by the dialog details and the button name should be read at one go.
So please fix the issue.
URL: https://fundamental-ngx.netlify.app/#/core/popover
JAWS 2021

Thanks and Regards
Sanjay

**Comment by Lokanathan-k** [comment]
hi @Sanjay-1312 

Netlify URL is not updated with the latest build can u please test with the PR url
https://deploy-preview-5459--fundamental-ngx.netlify.app/#/core/dialog

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The dialog details is read first and the dialog name is read twice to  the user.
So please fix the issue.

The dialog name  followed by the dialog information should be read and the focused UI element name.
Screenshot:
![image](https://user-images.githubusercontent.com/78714518/119928538-64625700-bf99-11eb-9852-f8c7f21399f7.png)


Thanks and Regards
Sanjay

**Comment by puru-hk** [comment]
@Sanjay-1312. Coud you please retest this issues using this url https://deploy-preview-5682--fundamental-ngx.netlify.app/#/core/dialog. I have tested please find the below screenshot.

![image](https://user-images.githubusercontent.com/32538291/122032077-edc4b680-cdec-11eb-9abe-8b83bc38a358.png)

**Comment by platon-rov** [comment]
@Betrozov is it fixed? I've just tested on staging - seems like information still not read.

**Comment by platon-rov** [comment]
I see that Dialog read as expect using JAWS, but it's not read correctly using any other screen reader (VoiceOver on Mac). Since we only support JAWS closing this issue is reasonable, in case of any other issues new tickets should be created.
